exonum-dictionary.txt

```diff
@@ -15,6 +15,7 @@ byteorder
 checkpointed
 clippy
 clonned
+compat
```

Comment:
This is the thing that I would never like in rust: short names. :disappointed:

Reply:
That's from the `failure` crate. But yeah, `TyCtxt` level of compression of `rustc` deferentially makes code harder to read....